### ISSUE TITLE
Exposure models -- drug, device, environmental

### ISSUE THREAD

**Issue by Sigfried** [RTIInternational/NHLBI-BDC-DMC-HM#17]
Maybe these should be combined, but will start with them being separate.

- [x] Drug exposure
- [ ] Device exposure
- [ ] Environmental exposure (no OMOP example to follow, will hold off on this for now)

Strategy will be to copy an existing entity in the big yml file and adapt while using OMOP [drug](https://ohdsi.github.io/CommonDataModel/cdm53.html#drug_exposure) or device exposure documentation as a reference.

- [ ] Consider combining all these under an abstract Exposure class/entity, but not for now.

**Comment by Sigfried** [comment]
Adding some notes as I formulate the DrugExposure entity model.

I am copying from the Condition entity and making changes from there.

- Noticed that the `associated_visit` attribute in Condition model referred to `QuestionnaireResponse`, which I assume is a mistake and fixed it.
- Condition has a `relationship_to_participant` attribute which I think is probably not appropriate to exposure. Not including it.
- [OMOP 5.3 drug_exposure](https://ohdsi.github.io/CommonDataModel/cdm53.html#drug_exposure) has a `stop_reason` field, but it's documented as being marked for retirement in v6.0. Not including.
- Looking at other attributes in the yaml file, sometimes `multivalued: false` is specified, other times not. I assume that leaving the field out is fine if multivalued is not expected.
-

**Comment by bfurner** [comment]
Thanks.

- yes, please clean up that reference in the description.  it looks like the description of a number of `DrugExposure` attributes retain verbiage referring to "condition" which we'll want to clean up to mention "exposure" instead at some point. 
- i agree with not including `relationship_to_participant`
- i agree with not including `stop_reason`
- i believe the default value for `mutivalued` is false, so yes this isn't explicitly necessary

**Comment by Sigfried** [comment]
@bfurner, I'm not sure what reference you're saying I should clean up in your first bullet.

I've just commited https://github.com/RTIInternational/NHLBI-BDC-DMC-HM/pull/24/commits/ea25d56882c07b88960df8aae763fd41a59c3f2c, fixing the references to "condition" in the DrugExposure entity.

I also added some comments asking where to get enum values from. In ConditionConceptEnum, for instance, the values come from HPO and MONDO. I've suggested getting the drug enums from OMOP, but only because OMOP is what I know. I don't know if it makes sense to have source enum values coming from such different places. Maybe I'll bring this up in the meeting happening in a bit.
